sphinxcontrib/datatemplates/loaders.py

```diff
@@ -99,14 +131,13 @@ def load_yaml(source,
             yield yaml.safe_load(f)
 
 
-@append_loader("xml",
-               (lambda source: "xml" in mimetypes.guess_type(source)[0]))
+@lenient_mimetype_loader('xml', 'xml')
 @contextlib.contextmanager
 def load_xml(source, absolute_resolved_path, **options):
     yield ET.parse(absolute_resolved_path).getroot()
 
 
-@append_loader("dbm")
+@file_extension_loader("dbm", ['.dbm'])
```

Comment:
It seems there is no file extension for DBM databases as they normally consist of multiple files. See https://en.wikipedia.org/wiki/DBM_(computing)#cite_note-2001-ladd-odonell-xhtml-3